Tests.Unit.Presentation.Web/DomainServices/QueryByRightsHolderIdsTest.cs

```diff
@@ -1,81 +0,0 @@
-﻿using Core.DomainModel.ItSystem;
-using Core.DomainServices.Queries.Interface;
-using System.Collections.Generic;
-using System.Linq;
-using Tests.Toolkit.Patterns;
-using Xunit;
-
-namespace Tests.Unit.Presentation.Web.DomainServices
-{
-    public class QueryByRightsHolderIdsTest : WithAutoFixture
```

Comment:
Denne test er for interfaces og ikke it systemer. Er det meningen den skulle fjernes?

Reply:
ugh.. det tror jeg ikke. Der gik merge amok